File: my-folder/2232-adding-spaces-to-a-string/solution.cpp
class Solution {
public:
    string addSpaces(string s, vector<int>& spaces) {
    string result = "";
    int j=0;
    for (int i = 0; i < s.length(); i++) {
        if(j<spaces.size() && spaces[j] == i) {
            result += ' ';
            j++;
        }
    result += s[i];
    }
    return result;
    }
};

File: my-folder/2232-adding-spaces-to-a-string/solution.js
/**
 * @param {string} s
 * @param {number[]} spaces
 * @return {string}
 */
var addSpaces = function(s, spaces) {
    var result = "";
    var j = 0;
    for(var i=0;i<s.length;i++){
        if(spaces[j]==i){
            result+=' ';
            j++;
        }
        result+=s[i];
    }
    return result;
};
